docs/src/pages/guides/FHIRBulkOperations.md

```diff
@@ -0,0 +1,193 @@
+The `$export` operation uses three OperationDefinition: 
+- [System](http://hl7.org/fhir/uv/bulkdata/STU1/OperationDefinition-export.html) - Export data from the server. Exports to an S3-compatible data store.
+- [Patient](http://hl7.org/fhir/uv/bulkdata/STU1/OperationDefinition-patient-export.html) - Obtain a set of resources pertaining to all patients. Exports to an S3-compatible data store.
+- [Group](http://hl7.org/fhir/uv/bulkdata/STU1/ OperationDefinition-group-export.html) - Obtain a set resources pertaining to patients in a specific Group. Only supports static membership; does not resolve inclusion/exclusion criteria.
+
+### **$export: Create a Bulk Data Request**
+To create an import request, the IBM FHIR Server requires the body fields of the request object to be a FHIR Resource `Parameters` JSON Object.  The request must be posted to the server using `POST`. Each request is limited to a single resource type in each imported or referenced file.
+
+#### Example Request - GET
+The following is a request to export data to the IBM COS endpoint from the IBM FHIR Server using GET.
+
+```sh
+curl -k -u "fhiruser:change-password" -H "Content-Type: application/fhir+json" -X GET 'https://localhost:9443/fhir-server/api/v4/$export?_outputFormat=application/fhir%2Bndjson&_type=Patient' -v
+```
+
+#### Example Request - POST
+The following is a request to export data to the IBM COS endpoint from the IBM FHIR Server using POST and FHIR Resource Parameters.
+
+```sh
+curl -k -u "fhiruser:change-password" -H "Content-Type: application/fhir+json" -X POST 'https://localhost:9443/fhir-server/api/v4/$export' -d '{
+    "resourceType": "Parameters",
+    "id": "e33a6a4e-29b5-4f62-a3e9-8d09f50ae54d",
+    "parameter": [
+        {
+            "name": "_outputFormat",
+            "valueString": "application/fhir+ndjson"
+        },
+        {
+            "name": "_since",
+            "valueInstant": "2019-01-01T08:21:26.94-04:00"
+        },
+        {
+            "name": "_type",
+            "valueString": "Observation,Condition"
+        }
+    ]
+}'
+```
+
+## Import Operation: $import
+
+The `$import` operation is System operation are invoked at `[base]/$import`.  The Import Operation uses a custom crafted OperationDefinition [link](https://github.com/IBM/FHIR/blob/master/fhir-operation-bulkdata/src/main/resources/import.json), which follows the proposal from [Smart-on-FHIR: import.md](https://github.com/smart-on-fhir/bulk-import/blob/master/import.md).
+
+### **$import: Create a Bulk Data Request**
+To create an import request, the IBM FHIR Server requires the body fields of the request object to be a FHIR Resource `Parameters` JSON Object.  The request must be posted to the server using `POST`. Each request is limited to a single resource type in each imported or referenced file.
+
+The IBM FHIR Server limits the number of inputs per each `$import` request based on `fhirServer/bulkdata/maxInputPerRequest`, which defaults to 5 input entries.
```

Comment:
I still think it would be nice if we supported many more inputs in a request and had the smarts to just do 5 at a time... do we have an issue open for that?

Reply:
Yes - https://github.com/IBM/FHIR/issues/948 - my thought is to split them into separate job paramters

Reply:
perfect, just wanted to make sure we were tracking it.